controllers/logsController.js

```diff
@@ -0,0 +1,85 @@
+const express = require("express");
+const logs = express.Router();
+const logsModel = require("../models/log");
+
+const logsArr = [...logsModel];
```

Comment:
This line is the problem as to why you're not passing the tests and also why your functionality will fail. The original array from the model folder is your "DB". If you make a copy and make changes on the copy than it's not persisting in the model array, and the changes will not persist between HTTP requests